### DISCUSSION TITLE
Feature request: Apply distinct Time grains to multiple specific columns within a Single chart

### DISCUSSION THREAD

**Discussion by saravana-ideas2it** [apache/superset#25820] [Ideas]
Could you kindly offer guidance on applying distinct time grains to multiple specific columns within a single chart? Here's the situation: I'm in the process of creating a pivot table for vendor invoices and I intend to assign column-specific time grains to the following fields:

InvoiceQuarter -> Time Grain: "Quarter"
PaymentMonthYear -> Time Grain: "Month"
Nevertheless, I'm encountering difficulties when attempting to simultaneously apply column-specific time grains to both of these columns. I'd greatly appreciate your support in addressing this scenario.

Please advise if Jinja templates can be used to achieve this.

<img width="955" alt="image" src="https://github.com/apache/superset/assets/129499348/0de9d080-32e9-4b01-b53a-51e2cc450577">

**Comment by rusackas** [comment]
This sounds more like a feature request than a SIP issue, so I'll go ahead and migrate this Issue over to a discussion. Please feel free to reach out to people on [Superset Slack](http://bit.ly/join-superset-slack) as well.

**Reply by saravana-ideas2it** [reply]
Thank you @rusackas for the update. Is there a possibility to prioritize this feature?

**Reply by rusackas** [reply]
I have no power to make any org/company take on a feature as a priority. If you're open to making a proposal for the design and/or implementation, folks would be happy to discuss/review any proposal or pull request(s).
